### ISSUE TITLE
Provide "InfoBox" annotations as param comments

### ISSUE THREAD

**Issue by WittleWolfie** [WittleWolfie/WW-Blueprint-Core#39]
E.g.:

```cs
[InfoBox("Fact that holds action should be the reason rather than attack roll from context (to prevent infinite cycles")]
public bool SetFactAsReason;
```

The parameter should have that info as a comment.

**Comment by WittleWolfie** [comment]
Sometimes it's "Tooltip"

**Comment by WittleWolfie** [comment]
This will launch w/ v2.0 and includes ComponentName which often provides a description of component / action / condition behavior.

Note that it's not feasible to use GetCaption() since it relies on the configuration of the object.

**Comment by WittleWolfie** [comment]
Implemented in 2.0.
